hal/src/sercom/v2.rs

```diff
@@ -40,6 +40,14 @@ pub mod uart;
 #[cfg(feature = "dma")]
 pub mod dma;
 
+/// Register synchronization flags
+pub enum reg_sync {
```

Comment:
I think enums are usually camel case `RegSync`

Reply:
I would like to look at this and verify it's the best approach to integrating with the rest of the `sercom` code.